### ISSUE TITLE
Codes and hyperparameters to reproduce the experiments

### ISSUE THREAD

**Issue by hkiyomaru** [wyshi/Unsupervised-Structure-Learning#1]
Could you share codes which were used for the experiments?
`main.py` does not select model parameters based on the performance on the validation set (the codes are commented out). It is different from the fact explained in the paper.
Additionally, I would like you to share hyperparameters which should have been selected according to the validation score.
Most hyperparameters are written in `config_utils.py`, but the file includes a lot of lines which are commented out. Which parameters were specified? What values were selected?
Thanks.

**Comment by dmlist** [comment]
I have the same question. Did you had solved it?  How to set hyperparameters?

**Comment by wyshi** [comment]
Hello, here is a pytorch implementation with all the parameters. 
https://github.com/Liang-Qiu/SVRNN-dialogues
